tests/test_raster.py

```diff
@@ -108,6 +108,17 @@ def test_arr(self):
         self.assertEqual(img_last.arr.shape, (385, 502, 1))
         img_last.close()
 
+    def test_set_array(self):
+        img = Image(TEST_FILE, dimorder="first")
+        img.arr = img.arr + 1
+
+        self.assertEqual(np.sum(img.arr), np.sum(np.ones(shape=img.arr.shape)))

```

Comment:
Why don't you test whether the arrays are the same? TEST_FILE is an array with 0s, correct?

Reply:
You are right. That's much clearer.